src/evaluation.py

```diff
@@ -0,0 +1,79 @@
+# src/evaluation.py
+
+import logging
+import matplotlib.pyplot as plt
+from sklearn.metrics import classification_report, accuracy_score
+
+logger = logging.getLogger(__name__)
+logging.getLogger("matplotlib.category").setLevel(logging.WARNING)
+
+
+def evaluate_model(model, x_test, y_test) -> plt.Figure:
```

Comment:
We could separate evaluation and plotting of the results into separate functions. The evaluation could return the classification report and then we could have another method for creating the plots.